source/documentation/appendix-docs/conda.md

```diff
@@ -139,11 +114,24 @@ environment.
 conda env update -f environment.yml --prune
 ```
 
-[packrat]: https://rstudio.github.io/packrat/
-[anaconda search]: https://anaconda.org/search
-[conda cheat sheet]:
-  https://conda.io/projects/conda/en/latest/user-guide/cheatsheet.html
 
-```{r global_options, include=FALSE}
-{knitr::opts_chunk$set(eval = FALSE)}
+## Shiny
+To use conda in shiny applications, you will need to use a different dockerfile to deploy the app. The [conda branch of the rshiny-template](https://github.com/moj-analytical-services/rshiny-template/tree/conda) has an appropriate dockerfile for this purpose. This is also necessary if you wish to use Python in your Shiny application, including using `dbtools` for accessing Athena databases.
+
+## Platform limitations
+
+There are a number of limitations and pitfalls to conda management to be aware of.
+
+### R package  versions on conda
+
+While Anaconda hosts most R packages on CRAN, some R packages on Anaconda only have binaries built for certain versions, and don't have a version for `R 3.5.1` as is currently used on the Analytical Platform. You can identify the available versions by inspecting the first few characters of the Build part of the filename - for example, **r351**h96ca727.
```

Comment:
It might be good to make the language less technical here. Can you possibly link or in some way let people see where to read the build part of the filename, too?

Also the asterisks are playing merry hell with your markdown, resulting in the following stuff all being bolded.

Reply:
Probably best to use \* to avoid this

Reply:
had to backslash my backslash to get a single backslash to show

Reply:
Revised with some handy images